mw-webapp/src/styles/_variables.scss

```diff
@@ -42,6 +42,7 @@ $tooltipZIndex: 110;
 $dropdownZIndex: 110;
 $modalZIndex: 120;
 $progressIndicatorZIndex: -1;
+$tooltipZIndex: 120;
```

Comment:
But we have already the same var on line 41.

Reply:
The error was not knocked out by the repetitive style. Fixed the problem and changed it to 10 indexes higher. I tried 110, it turned out that the tooltip had a lower priority. Therefore, I made it higher